app/uk/gov/hmrc/traderservices/journeys/TraderServicesFrontendJourneyModel.scala

```diff
@@ -48,10 +48,11 @@ object TraderServicesFrontendJourneyModel extends JourneyModel {
     def isVesselDetailsAnswerMandatory(importQuestions: ImportQuestions): Boolean =
       importQuestions.requestType.contains(ImportRequestType.Hold)
 
-    val maxFileUploadsNumber = 3
+    val maxFileUploadsNumber: Int = 10
```

Comment:
The actual change here is fine although I did notice that if you click continue after adding the max amount of files we get to a file not found/work in progress page. I messed about with this number locally to test and it seems to repeat this behaviour each time but only when it's the max number (the file types etc are allowed)

Reply:
good catch

Reply:
fixed